CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## tip
 
+* FEATURE: updated `localStorage` keys to improve uniqueness. See [this PR](https://github.com/VictoriaMetrics/victoriametrics-datasource/pull/259).
```

Comment:
```suggestion
* FEATURE: change `localStorage` key names to avoid collisions with other systems. See [this PR](https://github.com/VictoriaMetrics/victoriametrics-datasource/pull/259).
```